package.json

```diff
@@ -0,0 +1,31 @@
+{
+  "name": "fluxible-plugin-copy-text",
+  "version": "1.0.0",
+  "description": "Fluxible plugin for using copy-text as a context method",
+  "author": "1stdibs.com, Inc.",
+  "scripts": {
+    "test": "mocha",
+    "gitlint": "node_modules/.bin/dibslint --git --warnings",
+    "lint": "dibslint --root=."
+  },
+  "main": "fluxible-plugin-copy-text.js",
+  "pre-commit": {
+    "run": [
+      "gitlint"
+    ]
+  },
+  "devDependencies": {
+    "dibslint": "^1.2.4",
+    "mocha": "^2.3.4",
+    "nodemon": "^1.8.1",
+    "pre-commit": "^1.1.2"
+  },
+  "dependencies": {
+    "copy-text": "^1.1.0",
+    "fluxible": "^1.0.3",
+    "fluxible-addons-react": "^0.2.2",
+    "lodash.assign": "^3.2.0",
+    "react": "^0.14.5",
+    "react-dom": "^0.14.5"
```

Comment:
probably but peerDependencies are being removed in npm 3+ so I didn't want to add it but I guess it won't break in later version of npm I think?